index.html

```diff
@@ -416,6 +416,17 @@
     return result;
 }
 
+function removeTower(x, y, undo = false){
+    for (let i = 0; i < towers.length; i++) {
+        const tower = towers[i];
+        if (tower.gridX === x && tower.gridY === y) {
+            const [removedTower] = towers.splice(i, 1);
+            money += tower.cost;
+            if (!undo) undoList.push(() => placeTower(x, y, 0, true));
+        }
```

Comment:
add "return true" if a tower was removed and "return false" at the end if nothing was removed